src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/util/AuthorizationEnabledIntegrationTest.java

```diff
@@ -405,6 +413,12 @@ public String retrieveSuperUserIdFromOrganisationId(String orgId) {
         return users.get(0).getId().toString();
     }
 
+    public String retrieveSuperUserIdentifierFromOrganisationId(String orgId) {
+        Organisation organisation = organisationRepository.findByOrganisationIdentifier(orgId);
+        List<ProfessionalUser> users = professionalUserRepository.findByOrganisation(organisation);
+        return users.get(0).getUserIdentifier();
```

Comment:
do we need to safeguard this? any possibility of Exceptions (IndexOutOfBoundsException)

Reply:
i don't think so, as every `organisation` must have at least a `superuser` with it. so the `users.get(0).getUserIdentifier();` should always return at least the `superuser`. see also above method which does something similar